admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java

```diff
@@ -189,14 +186,40 @@ public <T> T getForObject(String url,
 
             try {
                 restTemplate = getRestTemplate();
-                result = (T) restTemplate
+                result= (T) restTemplate
                         .getForObject(url, responseType);
             } catch (Exception e) {
                 throw new Exception(e);
             }
         return result;
     }
 
+    /**
+     * Gets the Object.
+     *
+     * @param <T>             the generic type
+     * @param url         the url
+     * @param responseType    the response type
+     * @return the api
+     * @throws Exception
+     */
+    @SuppressWarnings("unchecked")
+    public <T> T getForApi(String url,
```

Comment:
why do we need this extra method, if get for object takes the same arguments ?

Reply:
we need token here

Reply:
and its commonly used method so i have created another one

Reply:
I am not ok with the name, but for now it is fine